handbook/product/product_management/responding_to_user_feedback.md

```diff
@@ -0,0 +1,21 @@
+# Responding to user feedback 
+
+The product team owns a number of [user feedback sources](../user_feedback.md). 
+
+## Feedback rotation
+
+A single product manager owns the general feedback channels for 2-4 weeks at a time, and then rotates this responsibility to the next-in-line product manager. 
+
+The [feedback workflow doc](https://docs.google.com/document/d/1TTRjK-CL38fdCvrVUgRL70agUiwDbQFJXCo8IuJmLls/edit#) lists the current responder. 
+
+### NPS Feedback
+
+> _TODO copy over some stuff from doc_
```

Comment:
I think we can move most of the doc here, except for the concrete templates (which might result in a weird user experience). If it's too much content we can move it out to a separate page of its own.

Reply:
Great, will do.

Reply:
@Joelkw I just got off a call with @virginiaulrich about how we do this process, so we can maybe hold off on adding that section just yet if she feels like we might change it soon

Reply:
Ah, I also spoke with her last week! I'll take a closer look and link to the doc for most of it / just pull out a few things that make sense to have high-level.